data/src/main/java/com/microsoft/azure/kusto/data/KustoResultSetTable.java

```diff
@@ -297,7 +298,7 @@ public byte getByte(String columnName) {
     }
 
     public short getShort(String columnName) {
-        return (short) getShort(findColumn(columnName));
+        return getShort(findColumn(columnName));
```

Comment:
Why aren't the others also implemented this way?
```java
public int getInt(String columnName) {
        return getInt(findColumn(columnName));
        }
```

Reply:
I looked briefly, and it looks like their call stacks are the same in practice, as `private Object get(String colName)` calls `public int findColumn(String columnName)`. It's just inconsistent how sometimes the `get[DataType]` method calls `findColumn` directly, and sometimes calls it via `get`.